megamek/src/megamek/client/ui/dialogs/BVDisplayDialog.java

```diff
@@ -41,25 +43,49 @@ public BVDisplayDialog(final JFrame frame, final boolean modal, final Entity ent
         this.entity = Objects.requireNonNull(entity);
         initialize();
     }
-    //endregion Constructors
 
-    //region Getters
+    @Override
+    protected void finalizeInitialization() {
+        super.finalizeInitialization();
+        setTitle(getTitle() + " (" + entity.getShortName() + ")");
+        UIUtil.adjustDialog(getContentPane());
+        pack();
+        Dimension screenSize = UIUtil.getScaledScreenSize(this);
+        setSize(new Dimension(getSize().width, Math.min(getHeight(), (int) (screenSize.getHeight() * 0.8))));
+    }
+
     public Entity getEntity() {
         return entity;
     }
-    //endregion Getters
 
-    //region Initialization
     @Override
     protected Container createCenterPane() {
-        final JEditorPane editorPane = new JEditorPane("text/html", getEntity().getBVText());
-        editorPane.setEditable(false);
-        editorPane.setCaretPosition(0);
-        editorPane.setFont(UIUtil.getScaledFont());
+        FlexibleCalculationReport bvReport = new FlexibleCalculationReport();
+        entity.calculateBattleValue(bvReport);
+
+        JButton exportText = new JButton("Copy as Text");
+        exportText.addActionListener(e -> copyToClipboard(bvReport.getTextReport().toString()));
+        JButton exportHTML = new JButton("Copy as HTML");
+        exportHTML.addActionListener(e -> copyToClipboard(bvReport.getHtmlReport().toString()));

```

Comment:
Given how we shorten entity to e everywhere across the project, can we please use evt to ensure it's obviously different?